### PR TITLE
refactored two img elements to one

### LINKED ISSUE #34: refactor double usage of `<img>` tag for event details
## Summary
Coming out of #32 , two `` tags where used to enable the feature of having the RI map achieve different outcomes for smaller vs larger breakpoints, with usage of positioning.
https://github.com/AnalogStudiosRI/www.blissfestri.com/blob/main/src/components/event-details/event-details.js

## Details
In theory this could / should be refactored down into one `` tag, and ideally all the styling leveraging Tailwind styles.

### PR DESCRIPTION
**Pull request by Udassi-Pawan**
Resolves #34 

## Summary of Changes
I refactored the two img elements to just one conserving the styles.
I am relatively newer to open source contributions. Do let me know if any changes are required.

### CODE REVIEW

File: src/components/event-details/event-details.js [hunk 1]
```diff
@@ -4,7 +4,9 @@ export default class EventDetails extends HTMLElement {
     this.innerHTML = `
       <div class="inline-block event-details">
 
-        <img src="/assets/images/ri-map.webp" class="hidden sm:block max-w-sm md:max-w-sm lg:max-w-md xl:max-w-md" style="position:absolute; z-index:-1">
+
+        <img src="/assets/images/ri-map.webp" class="block pt-2 w-350 sm:pt-0 sm-max-w-sm md:max-w-sm lg:max-w-md xl:max-w-md sm:absolute sm:z-(-1)">
```
Comment (Arindam200) [inline src/components/event-details/event-details.js:8]:
```suggestion
        <img src="/assets/images/ri-map.webp" class="block pt-2 w-[350px] sm:pt-0 sm-max-w-sm md:max-w-sm lg:max-w-md xl:max-w-md sm:absolute sm:z-(-1)">
```

Comment (Arindam200) [reply src/components/event-details/event-details.js:8]: I think w-350 will not work. Instead, you can use `w-[350px]`

**Review by Arindam200 — CHANGES_REQUESTED** [review]

File: src/components/event-details/event-details.js (hunk 1 above)
Comment (Arindam200) [inline src/components/event-details/event-details.js:8]:
```suggestion
        <img src="/assets/images/ri-map.webp" class="block pt-2 w-350 sm:pt-0 sm-max-w-sm md:max-w-sm lg:max-w-md xl:max-w-md sm:absolute sm:z-[-1]">
```

Comment (Arindam200) [reply src/components/event-details/event-details.js:8]: And I think it should be `z-[-1]`

**Review by Arindam200 — CHANGES_REQUESTED** [review]

**Review by thescientist13 — COMMENTED** [review]
Thanks @Udassi-Pawan for this!

I realize I forgot to leave out an important detail in the issue 🤦‍♂️ 

So for mobile, we needed to use a second image where the dots keep going down to the logo just because we wanted to keep everything stacked and narrow for these kind of screen sizes.  I have a PR up for this now that I got the latest image from the designer - #40 
Screen Shot 2023-04-24 at 7 10 11 PM

This means that we would still need to manage two different `src` attributes based on breakpoint.  I'm thinking we could use [`srcset`](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images#resolution_switching_different_sizes) to select which image to load, while still keeping the styles all in Tailwind?  Hopefully that allows us to still use one `` tag.  🤞 

Let me know your thoughts, and apologies for the missing details.

**Comment by thescientist13** [comment]
Just wanted to share an example that I just implemented for the header banner image that you can reference
https://github.com/AnalogStudiosRI/www.blissfestri.com/blob/main/src/components/header/header.js#L9
```html
  <img
    src="/assets/images/blissfest-logo-sm.webp"
    srcset="/assets/images/blissfest-logo-sm.webp 500w,
            /assets/images/blissfest-logo-md.webp 1000w,
            /assets/images/blissfest-logo-lg.webp 1500w"
  />
```

In this case I did the following:
1. Set the `src` to the default image to show, which in our case we want to target mobile as the default
1. Create an entry in the source set at each breakpoint I wanted to target; 350px, 1000px, 1500px across three different screen sizes.

So to apply that to this instance, I think something like this should probably work
```html
<img
  src="/assets/images/ri-map-mobile.webp"
  srcset="/assets/images/ri-map-mobile.webp 350w,
          /assets/images/ri-map.webp
/>
```

**Pull request closed without merging**